### PR TITLE
feat: messages about followings and userposts

### PR DESCRIPTION
**Pull request by ManuDiasCruz**

### CODE REVIEW

File: src/components/Timeline/index.jsx [hunk 1]
```diff
@@ -13,7 +13,8 @@ import Hashtags from "../Hashtags/index.jsx";
 export default function Timeline() {
   const statusMessages = {
     loading: "Loading",
-    emptArray: "There are no posts yet",
```
Comment (ManuDiasCruz) [inline src/components/Timeline/index.jsx:19]: We are not using this anymore, as we have to new messages to show about posts status

File: src/components/Timeline/index.jsx [hunk 2]
```diff
@@ -31,18 +32,23 @@ export default function Timeline() {
   }, [getContext().contextData]);
 
   function getPosts() {
+
     const promisse = axios.get(`${url}/posts${queryHashtag}`, config);
     promisse
       .then((response) => {
         const data = response.data;
-        if (data.length === 0) setPosts(statusMessages.emptArray);
```
Comment (ManuDiasCruz) [inline src/components/Timeline/index.jsx:49]: I had to change this if also for putting new verification clausure

File: src/components/Timeline/index.jsx [hunk 3]
```diff
@@ -31,18 +32,23 @@ export default function Timeline() {
...
-        else setPosts(data);
+        if (data == "-1")
+          setPosts(statusMessages.noFollowings);
+        else if (data.length === 0) 
+          setPosts(statusMessages.noPosts);
+        else 
+          setPosts(data);
       })
       .catch((error) => {
-        console.log(error.response);
```
Comment (ManuDiasCruz) [inline src/components/Timeline/index.jsx:71]: I took out to clean code

**Review by ManuDiasCruz — COMMENTED** [review]
fix: explaining my modifications

File: src/components/Timeline/index.jsx (hunk 1 above)
Comment (weslenmendes) [reply src/components/Timeline/index.jsx:19]: ok

File: src/components/Timeline/index.jsx (hunk 2 above)
Comment (weslenmendes) [reply src/components/Timeline/index.jsx:49]: ok

File: src/components/Timeline/index.jsx (hunk 3 above)
Comment (weslenmendes) [reply src/components/Timeline/index.jsx:71]: ok

**Pull request merged**